app/resources/views.py

```diff
@@ -0,0 +1,61 @@
+from flask import render_template, redirect, url_for
+from flask.ext.login import login_required, current_user
+from . import resources
+from .. import db
+from ..models import Resource, ZIPCode, Address, User
+from .forms import ResourceForm
+
+
+@resources.route('/', methods=['GET'])
```

Comment:
I don't think either is better style per se, so I guess it doesn't really matter. Matter of personal preference.